scripts/chainlist/result.txt

```diff
@@ -0,0 +1,457 @@
+DFK_CHAIN = 53935
```

Comment:
Maybe this file should be included in gitignore ?

Reply:
I included this file in gitignore at first, but I'm not sure if @Uxio0 was asking to add it [here](https://github.com/safe-global/safe-eth-py/pull/310#pullrequestreview-1052681431). Maybe I misunderstood his comment, I'm fine with both options, maybe we can gitignore it since there's no value in having it right now.

Reply:
I added back the file to gitignore, if @Uxio0 thinks this should be uploaded we could generate it anytime :)